### PR TITLE
EN_Reboot: Sept 20th Hotfixes

### PR DESCRIPTION
**Pull request by Frey8914**
-Fix Stellar Gift etrials and tutorial.
Standardized the Stellar Gift text and restored missing voice line playback. Fixed remove protection to disengage protection as the enemies are cleared to disengage the automatic protection rather than directly removing the protection.

-Fix Task descriptions.
Standardized the partying text.

-Import motions.
Imported motion names.

-Fix Mining Rig Defense.
Fixed formatting of incident text.

-Fix NGS potentials.
Fixed potentials.

-Restore tab spacing.
Restored tab spacings that were turned into \t.

Addendum
-Fix camera options.
Fixed the fixed camera text button guide that stated Up/Down and Left/Right as Horizontal typoed and Vertical.

### CODE REVIEW

**Comment by DabirSA** [comment]
Looks good to me.

File: Translated/Tutorials/tut_006371.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 Title#0,"""Stellar Gift Trials"""
-text0001#0,"""Stellar Gift Trials task you with investigating a \""Stellar Gift\"" that has fallen from the sky.<br><br>The fallen Stellar Gift will be protected, so to retrieve it you will need to remove that protection."""
+text0001#0,"""Stellar Gift Trials task you with investigating a \""Stellar Gift\"" that has fallen from the sky.<br><br>The fallen Stellar Gift will be protected, so to retrieve it you will need to disengage that protection."""
```
Comment (SynthSy) [inline Translated/Tutorials/tut_006371.csv]: That's not the proper way to use that word. Revert this.

Comment (DabirSA) [reply Translated/Tutorials/tut_006371.csv]: It's a completely fine way to use the word. Certainly looks in line with what we see in-game - we don't interact with the capsule's protection directly, we clear the enemies and it turns itself off.

**Review by SynthSy — CHANGES_REQUESTED** [review]
Just needs a small revision, otherwise all is well.

**Pull request merged**